plugin.rb

```diff
@@ -20,6 +20,7 @@ class Engine < ::Rails::Engine
 
     def self.clean_tag(tag)
       tag.downcase.strip[0...SiteSetting.max_tag_length].gsub(TAGS_FILTER_REGEXP, '')
+                                                        .gsub(/\.(json|rss)$/, '')
```

Comment:
I seem to have it working with the proper method now. I tried the same thing yesterday but couldn't seem to get it to work, today I got it to work.